stubs/TYPO3/CMS/Core/Http/UploadedFile.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace TYPO3\CMS\Core\Http;
+
+if (class_exists('TYPO3\CMS\Core\Http\UploadedFile')) {
+    return;
+}
+
+class UploadedFile
+{
+
+}
```

Comment:
new line at EOF missing

Reply:
i had cases where it just wasn't displayed in github? Maybe that's the issue?

Reply:
In this case it is really missing.

Reply:
Yes i got it enabled